test/skate_web/controllers/notification_read_states_controller_test.exs

```diff
@@ -104,24 +102,23 @@ defmodule SkateWeb.NotificationReadStatesControllerTest do
              ) == 2
 
       read_notification_user_ids =
-        Skate.Repo.all(
-          from(nu in DbNotificationUser,
-            select: {nu.notification_id, nu.user_id},
-            where: nu.state == ^read_state
+        Enum.sort(
+          Skate.Repo.all(
+            from(nu in DbNotificationUser,
+              select: {nu.notification_id, nu.user_id},
+              where: nu.state == ^read_state
+            )
           )
         )
```

Comment:
suggestion(non-blocking): what do you think about turning this into a multi-pipe again?
```suggestion
        from(nu in DbNotificationUser,
            select: {nu.notification_id, nu.user_id},
            where: nu.state == ^read_state
        )
        |> Skate.Repo.all()
        |> Enum.sort()
```

Reply:
Though, I would expect `Credo.Check.Readability.NestedFunctionCalls` to call this out...

Reply:
Done.

Perhaps I'll do a follow-up PR to enable `NestedFunctionCalls` as well.

Reply:
👍 Trying to think of how to do that while avoiding conflicts